Contants/Texts/Source/texts/Skills_BattleStatus.txt

```diff
@@ -851,3 +851,11 @@ L-Brace[X]
 ## MSG_SKILL_LunarBrace
 Lunar Brace:[NL]
 Negate 25% of enemy defense.[X]
+
+## MSG_SKILL_EffectSpore_NAME
+E-Spore[X]
+
+## MSG_SKILL_EffectSpore
+Effect Spore:[NL]
+If attacked, 50% chance of inflicting[NL]
```

Comment:
isn't this desc too long that may cause rtext overflow?

Reply:
![image](https://github.com/user-attachments/assets/594e9a16-ca9d-4f84-a372-69f782c254e5)